dictionary.lua

```diff
@@ -0,0 +1,22 @@
+Dictionary = Class('Dictionary')
+
+function Dictionary:initialize()
+  file = 'lib/dictionary.txt'
+  self.dict = {}
+  self.length = 0
+  for line in io.lines(file) do
+    self.length = self.length + 1
+    self.dict[self.length] = line
+  end
+  print('dict is ' .. self.length)
+end
+
+function Dictionary:get()
```

Comment:
these getters a redundant because an outside class can already access `dictObject.dict`, etc. If you wanted to hide `dict` or `length`, you'd need to set them as locals in the root scope of the dictionary cass